generate.coffee

```diff
@@ -105,7 +111,8 @@ generateOrders = (total, customers, addresses, products) ->
     createdAt = getRandomDate()
     customer = getCustomerToBuyFavoringRepeats(customers, orderCounts, createdAt)
     utmParameters = getUtmParameters()
-    
+    store = getRandomItem(stores)
```

Comment:
Actually, it looks like a customer can only have one store_id. I think we should have `store_id` be `customer.store_id`.